Ansible/ansible_collections/jfrog/platform/roles/postgres/templates/postgresql.conf.j2

```diff
@@ -97,7 +97,11 @@ max_connections = {{ postgres_server_max_connections }}			# (change requires res
 #ssl_key_file = 'server.key'		# (change requires restart)
 #ssl_ca_file = ''			# (change requires restart)
 #ssl_crl_file = ''			# (change requires restart)
+{% if fips_enabled %}
```

Comment:
You can use `{% if _fips_enabled.stdout == '1' }}`

Reply:
> You can use `{% if _fips_enabled.stdout == '1' }}`
My preference would be a boolean over a string comparison. Maybe use `{% if _fips_enabled.stdout | bool }}`?